### ISSUE TITLE
Arabic text detection issue with different font sizes

### ISSUE THREAD

**Issue by omantawy** [VikParuchuri/surya#28]
I tested with some Arabic text that has a title and some normal font size text. The model was not able to detect the title and it detected the normal size text missing out some words.
Check the screenshot below:

![Screenshot from 2024-02-14 09-39-21](https://github.com/VikParuchuri/surya/assets/14849202/15eeb1d7-28aa-42c7-b1f1-f3316b895d81)

**Comment by VikParuchuri** [comment]
Yes, this is an issue I've seen.  I'm working on fixing it now (retraining the model).

**Comment by mahavivo** [comment]
there is a minor issue with chinese OCR. run "surya_ocr DATA_PATH  --langs zh", you will get unicode instead of plain text of chinese.

**Comment by VikParuchuri** [comment]
> there is a minor issue with chinese OCR. run "surya_ocr DATA_PATH --langs zh", you will get unicode instead of plain text of chinese.

I'll merge something to make the output a little nicer.  But it doesn't matter either way for reading the json file
